pyworkflow/gui/form.py

```diff
@@ -2146,16 +2149,18 @@ def save(self, e=None):
 
     def schedule(self):
         if self.protocol.useQueue():
-            if not self._editQueueParams():
-                return
+            if not self.protocol.getQueueParams()[0]:
```

Comment:
I'd have a "queueReady()"? method to wrap queue logic?:

````
if self.protocol.useQueue():
            if not self.protocol.getQueueParams()[0]:
            if not self._editQueueParams():

Reply:
Yes!!! I'll create the method!!